Dockerfile

```diff
@@ -24,8 +24,9 @@ RUN apt-get -y update && apt-get install -y \
 
 RUN mkdir -p /grunt
 WORKDIR /
-RUN wget https://github.com/QIICR/dcmqi/releases/download/v1.0.5/dcmqi-linux.tar.gz
-RUN tar -xvzf dcmqi-linux.tar.gz -C /grunt
+RUN wget -O dcmqi-linux.tar.gz --quiet https://github.com/QIICR/dcmqi/releases/download/latest/dcmqi-1.0.5-linux-20171124-3f6534e.tar.gz
```

Comment:
You probably want to update this to use the latest tagged release - "latest" packages are not persistent!

https://github.com/QIICR/dcmqi/releases/tag/v1.0.9